public/app/features/dashboard/dashgrid/DashboardGrid.tsx

```diff
@@ -293,69 +294,69 @@ export class DashboardGrid extends PureComponent<Props, State> {
     }
   };
 
+  private resizeObserver?: ResizeObserver;
+  private rootEl: HTMLDivElement | null = null;
+  onMeasureRef = (rootEl: HTMLDivElement | null) => {
+    if (!rootEl) {
+      if (this.rootEl && this.resizeObserver) {
+        this.resizeObserver.unobserve(this.rootEl);
+      }
+      return;
+    }
+
+    this.rootEl = rootEl;
+    this.resizeObserver = new ResizeObserver((entries) => {
+      entries.forEach((entry) => {
+        this.setState({ width: entry.contentRect.width });
+      });
+    });
+
+    this.resizeObserver.observe(rootEl);
+  };
+
   render() {
     const { isEditable, dashboard } = this.props;
+    const { width } = this.state;
 
     if (dashboard.panels.length === 0) {
       return <DashboardEmpty dashboard={dashboard} canCreate={isEditable} />;
     }
 
-    /**
-     * We have a parent with "flex: 1 1 0" we need to reset it to "flex: 1 1 auto" to have the AutoSizer
-     * properly working. For more information go here:
-     * https://github.com/bvaughn/react-virtualized/blob/master/docs/usingAutoSizer.md#can-i-use-autosizer-within-a-flex-container
-     *
-     * pos: rel + z-index is required to create a new stacking context to contain the escalating z-indexes of the panels
-     */
+    const draggable = width <= config.theme2.breakpoints.values.md ? false : isEditable;
+
+    // pos: rel + z-index is required to create a new stacking context to contain
+    // the escalating z-indexes of the panels
     return (
       <div
+        ref={this.onMeasureRef}
         style={{
           flex: '1 1 auto',
           position: 'relative',
           zIndex: 1,
           display: this.props.editPanel ? 'none' : undefined,
         }}
       >
-        <AutoSizer disableHeight>
```

Comment:
@joshhunt this change to useMeasure breaks resizing a window and having the dashboard response to change, for that you need the container to have position absolute, that is at least the only solution I have found

Reply:
hm.. might have been something with a branch I was on, strange, can't replicate in main